src/braket/circuits/gates.py

```diff
@@ -90,7 +90,7 @@ def fixed_qubit_count() -> int:
         return 1
 
     @staticmethod
-    @circuit.subroutine(register=True)
+    @circuit.subroutine(register=True, cls_docstring=__doc__)
```

Comment:
Did you mean to only add this to the `h` gate, or do you also want to add this to the rest of the gates?